content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -1066,6 +1068,33 @@ model User {
 </tab>
 </TabbedContent>
 
+##### Define a scalar list with a default value
```

Comment:
Would be good to add that this was added in Prisma 4.0.0

Currently looks like this
<img width="564" alt="Screen Shot 2022-06-27 at 15 26 33" src="https://user-images.githubusercontent.com/1328733/175952899-3085c21e-9350-4afa-afcc-2d93b4dd9e8c.png">

Reply:
Thanks, done